docker-compose.yml

```diff
@@ -63,64 +63,68 @@ services:
     # small.
     command: mysqld --bind-address=0.0.0.0 --slow-query-log --log-output=TABLE --log-queries-not-using-indexes=ON
     logging:
-        driver: none
-
+      driver: none
+  # TODO(#6517): replace /test/redis-cluster.config with /test/redis.config on
+  # nodes 1-2.
```

Comment:
I believe this comment can/should be updated now that we're going the "just spin up 8 nodes in the interim" route?